docs/src/main/asciidoc/hibernate-panache.adoc

```diff
@@ -0,0 +1,1192 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Simplified Hibernate with Panache
+include::_attributes.adoc[]
+:categories: data
+:summary: Hibernate is the de facto Jakarta Persistence implementation and offers you the full breadth of an Object Relational Mapper. It makes complex mappings possible, but it does not make simple and common mappings trivial. Panache focuses on making your entities trivial and fun to write.
+:config-file: application.properties
+:topics: data,hibernate-orm,hibernate-reactive,panache,sql,jdbc
+:extensions: io.quarkus:quarkus-hibernate-panache,io.quarkus:quarkus-hibernate-orm,io.quarkus:quarkus-hibernate-reactive
+:hibernate-panache-api: https://javadoc.io/doc/io.quarkus.hibernate.panache/hibernate-panache/{quarkus-version}
+
+Hibernate with Panache is the perfect solution if you want to get started using Hibernate in Quarkus.
+
+WARNING: This extention is currently in preview. It is experimental, so its API may change, in particular names of
+classes or packages may change, even the module name. We are in the process of making it a public preview in order
+to obtain public feedback, so please report feedback either on link:https://quarkusio.zulipchat.com/#narrow/channel/187038-dev/topic/WG.20-.20Panache.2ENext/with/562916276[Zulip]
+or via link:https://github.com/orgs/quarkusio/projects/50/views/1[GitHub issues].
```

Comment:
This needs improvement IMHO. It's very unclear how it articulates with the existing extensions and we should clarify that.
And the name of the extension doesn't help. We go from `quarkus-hibernate-orm-panache` to `quarkus-hibernate-panache`, which is extremely confusing.

We really need to have a discussion on how to present this as I don't see how users won't be extremely confused.

Also we should use the `extension-status` include.

Let's tackle that after the break.

Reply:
We can improve. And the module name might change entirely. But the name is pretty coherent, because unlike `hibernate-orm-panache` and `hibernate-reactive-panache` this `hibernate-panache` just works with both. Though some people argue that the Hibernate brand is more general than ORM and HR, so perhaps that's what you're referring to.

I did not know about the `extension-status` include, I'll look at it.